index.html

```diff
@@ -231,6 +231,30 @@ <h3>Languages</h3>
         <img src="assets/vector.svg " alt="">
       </div>
   </section>
+
+  <section class="contact-me">
+    
+    <form>
+      <div class="background">
+      <h2>contact-me</h2>
+      <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+        I’d love to help with it!</p>
+      <label for="">
+        <input type="text" id="message" placeholder="Lorem Ipsum"/>
+      </label>
+      <label for="">
+        <input type="text" id="message" placeholder="loremipsum@gmail.com"/>
+      </label>
+      <textarea id="message-box" placeholder = "">
```

Comment:
Well done on adding the textarea for the message! 👍 However, there are a few adjustments that need to be made to meet the project requirements:

1. The placeholder for the message textarea is currently empty. Please consider adding a meaningful placeholder that provides clear instructions or examples of the expected input. This will help users understand the purpose of the field.

2. As per the project requirement, the message input should have the `maxlength` attribute set to `"500"`. This will limit the input to a maximum of 500 characters.

3. Additionally, the message input should have the `required` attribute added. This will activate the validation on form submission and ensure that the field is not left empty.

By adding a meaningful placeholder, setting the `maxlength` attribute to `"500"`, and including the `required` attribute for the message input, you will fulfill the project requirements. Keep up the great work! 👏🌟